store/store.go

```diff
@@ -548,7 +548,7 @@ func (s *Store) RemoveACI(key string) error {
 		dirUid := int(fi.Sys().(*syscall.Stat_t).Uid)
 
 		if uid != dirUid && uid != 0 {
-			return fmt.Errorf("permission denied, are you root or the owner of the image?")
+			return fmt.Errorf("permission denied, are you root or the owner of the image? User: uid=%d euid=%d. Image directory: uid=%d", uid, os.Geteuid(), dirUid)
```

Comment:
Yup sorry, I misread the context (although now I'm a bit perplexed by this uid-0 check).